solution/ui/regulations/utilities/utils.js

```diff
@@ -146,7 +161,12 @@ const getRequestParams = (query) => {
             );
 
             return filteredValues
-                .map((v) => `${PARAM_MAP[key]}=${v}`)
+                .map(
+                    (v) =>
+                        `${PARAM_MAP[key]}=${
+                            PARAM_ENCODE_DICT[key] ? encodeURIComponent(v) : v
```

Comment:
encode query string when assembling query params to be sent to `content-search` endpoint

Reply:
This is how we are creating the querystring for the `content-search` request on the Subjects page, which is a little different than how we do it on the Search page; however, the Search page is set to be refactored very soon and it will start using this same utility method then.